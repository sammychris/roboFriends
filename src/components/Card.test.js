// test file
import { shallow } from "enzyme";
import Card from "./Card";

const wrapper = shallow(<Card />);

it("expect to render Card component", () => {
  expect(wrapper.length).toEqual(1);
});

it("expect to make a snapshot to component", () => {
  expect(wrapper).toMatchSnapshot();
});
