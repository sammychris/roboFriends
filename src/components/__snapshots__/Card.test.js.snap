// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snapshot to component 1`] = `ShallowWrapper {}`;
